src/network.ts

```diff
@@ -0,0 +1,48 @@
+import { ethers } from 'ethers';
+import { BaseError } from './base';
+import { INFURA_API_KEY } from './config';
+
+export interface UnsupportedNetwork {}
+export class UnsupportedNetwork extends BaseError {}
+
+const NETWORK = {
+  LOCAL: 'local',
+  RINKEBY: 'rinkeby',
+  ROPSTEN: 'ropsten',
+  GOERLI: 'goerli',
+  MAINNET: 'mainnet',
+};
+
+export default function getNetwork(network: string): any {
+  let SUBGRAPH_URL: string;
+  let INFURA_URL: string;
+  switch (network) {
+    case NETWORK.LOCAL:
+      SUBGRAPH_URL = 'http://127.0.0.1:8000/subgraphs/name/graphprotocol/ens';
+      INFURA_URL = `https://rinkeby.infura.io/v3/${INFURA_API_KEY}`;
+      break;
+    case NETWORK.RINKEBY:
+      SUBGRAPH_URL =
```

Comment:
Can you add the comment that this version has the latest subgraph with NameWrapper support and to be replaced once the production version gets deployed?